Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -47,29 +45,62 @@ internal class RoutingTable
             {
                 _buckets[i] = new KBucket(_bucketSize, _random, _logger);
             }
-
-            _bucketMutex = new AsyncLock();
         }
 
         public int Count => _buckets.Sum(bucket => bucket.Count);
 
-        public IEnumerable<KBucket> NonFullBuckets
+        public IEnumerable<BoundPeer> Peers
+        {
+            get
+            {
+                var peers = new List<BoundPeer>();
+                foreach (KBucket bucket in NonEmptyBuckets)
```

Comment:
I guess we can use [`.SelectMany()`](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.selectmany?redirectedfrom=MSDN&view=netstandard-2.0#overloads).